nncf/tensorflow/graph/metatypes/common.py

```diff
@@ -79,9 +78,7 @@
 ]
 
 LINEAR_LAYER_METATYPES = [
-    layer_metatypes.TFDenseLayerMetatype,
-    # LINEAR_TF_OP_METATYPES
-    op_metatypes.TFMatMulOpMetatype,
+    layer_metatypes.TFDenseLayerMetatype
```

Comment:
Could you clarify if op_metatypes.TFMatMulOpMetatype operations will be quantized?